tutorials/dataframe/df004_cutFlowReport.C

```diff
@@ -58,13 +58,13 @@ void df004_cutFlowReport()
    auto filtered3 = augmented1.Filter(cut3, {"b3"}, "Cut3");
 
    // Statistics are retrieved through a call to the Report method:
-   // when Report is called on the main RDataFrame object, it retrieves stats
+   // When Report is called on the main RDataFrame object, it retrieves stats
```

Comment:
there is no capitalization after `:` in English?

```suggestion
   // when Report is called on the main RDataFrame object, it retrieves stats
```